_sections/COVID-19/en/07-it-is-here.md

```diff
@@ -1,19 +1,5 @@
-was detected in Seattle](https://twitter.com/trvrb/status/1236096904678633472). Seattle and Stanford are doing an amazing job of getting up to speed with their own testing kits; [about 5-7% of tests in Seattle are positive](https://twitter.com/UWVirology/status/1236017803162873856) and
-anyone (with a doctor's indication) can be tested.
+Assume that the virus is still in your city / town / workplace / church / etc. Your local testing capacity may vary, so it may not be possible to detect an outbreak in time to curtail it. However, not all venues are equally risky. During the strictest parts of the lockdown, infections tend to be localized to high-risk environments such as [care homes, prisons, and meat-packing facilities](https://www.vox.com/future-perfect/2020/5/1/21239396/covid-19-meatpacking-prison-jail-moral). As things open up, expect more infections to be in places where there are larger numers of people congregating indoors at one time, especially when there is also generational mixing, and an emphasis on close contact. 
 
-> Frustrated by the lack of testing resulting from the problem with the CDC-developed kit, the Seattle Flu Study began using an in-house developed test to look for COVID-19 in samples from people who had tested negative for flu. That work —permissible because it was research— uncovered the Snohomish County teenager.
-
-(For the sake of public health, University of Washington are publishing these results). So far only UW and Stanford have been moving ahead with
-their own (non-CDC) testing; both of these institutions have mandated that in-person classes be moved to distance learning alternatives. That
-speaks for itself; more universities should follow suit. That said, [professors need the technology and support to make this switch](https://twitter.com/ryanaboyd/status/1236009378295103488).
-
-Education communities that are resource-poor (without laptops or internet) would have the hardest time doing this switch. So let's start
-with those universities (and disciplines) for whom this switch is not such a major hardship. There is no one-size-fits-all intervention, but
-speed is key. Professors, please don't wait for your [university administration to make the decision campus wide](https://www.insidehighered.com/news/2020/03/06/roundup-weeks-news-about-colleges-and-coronavirus?utm_content=buffera0fc5&utm_medium=social&utm_source=linkedin&utm_campaign=IHEbuffer). Move to a distance-based option of your own accord. [Check out these tips for teaching online in a pinch](https://docs.google.com/document/d/1QR7IEgdisO6JtmELs07uUsSSu2Yox86GJY9wGV6mBjA/edit#).
+Of those who get infected, some people (aka 'superspreaders') readily transmit virus but others do not. The specific reasons for this are not yet well understood. If you happen to be in the presence of any such person, even an otherwise low-risk encounter might result in your getting infected.
```

Comment:
This is interesting and I was not aware of it. Do we have some data/evidence that super-spreaders are high-risk in all circumstances? My understanding was that some people (that shed more virus) engage in high-risk/high-spread-potential activities and infect very many people (e.g. going to big church, mingling in a crowd, etc).

Technically I can see how shedding large quantities of virus increases risk of infection in all circumstances but it was unclear to me that this trumps the low-risk environments.

Reply:
I should probably rephrase this.

Reply:
I can refine this further, but it is good enough for now